src/main/java/com/helloworld/springboot/HelloWorldController.java

```diff
@@ -9,11 +9,11 @@
 @Controller
 public class HelloWorldController {
 
-    @RequestMapping("helloworld")
+@RequestMapping("helloworld")
 
-    public String hello (ModelMap modelMap) {
-        modelMap.addAttribute ("message", "HelloWorld");
-        modelMap.addAttribute ("time", LocalDateTime.now ().toString ());
-        return "helloworld";
-    }
+public String hello (ModelMap modelMap) {
+    modelMap.addAttribute ("message", "HelloWorld");
+    modelMap.addAttribute ("time", LocalDateTime.now ().toString ());
```

Comment:
うまくフォーマットできていないですかね？
下記のようにメソッド名と`()`は間を空けないのが正しいです。
```suggestion
public String hello(ModelMap modelMap) {
    modelMap.addAttribute("message", "HelloWorld");
    modelMap.addAttribute("time", LocalDateTime.now().toString());
```